web/gds-user-ui/src/modules/dashboard/member/Components/MemberTableRows.tsx

```diff
@@ -0,0 +1,52 @@
+import { Td, Tr, Button, HStack, chakra } from '@chakra-ui/react';
+
+import UnverifiedMember from '../UnverifiedMember';
+import { BsEye } from 'react-icons/bs';
+
+interface MemberTableRowsProps {
+  rows: any;
+}
+
+const MemberTableRows: React.FC<MemberTableRowsProps> = (rows: any) => {
+  // for now we are just displaying the unverified member component
+  // with the status check story we will check if the member is verified or not and display the appropriate component
+  return (
+    <Tr>
+      {rows.length > 0 ? (
+        <>
+          <Td>
+            <chakra.span display="block"></chakra.span>
+            <chakra.span display="block" fontSize="sm" color="gray.700"></chakra.span>
```

Comment:
Could the 2nd span be removed? There will only be one name, so I think that it needs to be included.